### DISCUSSION TITLE
More StandardValueSets for manifest creation

### DISCUSSION THREAD

**Discussion by jh480** [forcedotcom/cli#2821] [Ideas]
Hello,

please consider adding the more recent GenderIdentity & Pronoun to the StandardValueSet selection from `sf project generate manifest`.

Thanks!

**Comment by mshanemc** [comment]
wanna send a PR? https://github.com/forcedotcom/source-deploy-retrieve/blob/da3c97b3453c7ef5df13b378f9d327fde16f95f5/src/registry/stdValueSetRegistry.json
